WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleAndStatusTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+final class TitleAndSubtitleAndStatusTableViewCell: UITableViewCell {
+
+    @IBOutlet private var subtitleLabel: UILabel!
+    @IBOutlet private var titleLabel: UILabel!
+    @IBOutlet private var statusContainerView: UIView!
+    @IBOutlet private var statusLabel: PaddedLabel!
+
+    static func register(for tableView: UITableView) {
+        tableView.registerNib(for: self)
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureBackground()
+        configureLabels()
+    }
+
+    func configureCell(viewModel: CellViewModel) {
+        titleLabel.text = viewModel.title
+        subtitleLabel.text = viewModel.subtitle
+        accessibilityLabel = viewModel.accessibilityLabel
+        statusLabel.text = viewModel.status
+        statusLabel.backgroundColor = viewModel.statusBackgroundColor
+    }
+
+}
+
+// MARK: - CellViewModel subtype
+//
+extension TitleAndSubtitleAndStatusTableViewCell {
+    struct CellViewModel {
+        var title: String
+        var subtitle: String
+        var accessibilityLabel: String
+        var status: String
+        var statusBackgroundColor: UIColor
```

Comment:
nit: just wondering if these could be `let`?

Reply:
Oh thank you, I missed this one 😅 Updated in 57ae49b205f4857e08cc6c6904593791067c5ae7 along with a change to the struct's naming to make it simpler.